app-frontend/src/app/pages/projects/edit/aoi-approve/aoi-approve.html

```diff
@@ -9,24 +9,77 @@ <h5 class="sidebar-title">AOI Scene Approval</h5>
     <li>
       <div class="label">
         <span class="text">
-          Select scenes to approve and click <strong>Approve Scenes</strong> to add to the project.
+          Select scenes to approve and reject and click <strong>Apply</strong> to add to the project. Any unselected scenes will be unaffected.
         </span>
       </div>
       <div class="btn-group fixedwidth">
         <button type="button"
                 class="btn btn-primary"
                 style="width: 12rem;"
-                ng-click="$ctrl.sceneModal()">
-          Approve Scenes
+                ng-click="$ctrl.applySceneStatuses()"
+                ng-disabled="!$ctrl.sceneStatusCounts.APPROVED && !$ctrl.sceneStatusCounts.REJECTED">
+          Apply
         </button>
         <button type="button"
                 class="btn dropdown-toggle"
-                ng-class="{'btn-secondary': $ctrl.allSelected}"
-                ng-click="$ctrl.selectAllScenes()">
-          <i ng-class="{'icon-plus': !$ctrl.allSelected,
-                       'icon-cross': $ctrl.allSelected}"></i>
+                ng-class="{'btn-secondary': $ctrl.sceneStatusCounts.APPROVED && $ctrl.sceneStatusCounts.APPROVED === $ctrl.sceneStatusCounts.TOTAL}"
+                ng-click="$ctrl.toggleAllSceneApproval()">
+          <i class="icon-check"></i>
         </button>
       </div>
     </li>
   </ul>
 </div>
+<div class="list-group" ng-if="$ctrl.pendingSceneList">
+  <div class="list-group-item">
+    <span class="no-margin" ng-if="!$ctrl.sceneStatusCounts.APPROVED && !$ctrl.sceneStatusCounts.REJECTED">No scenes selected to approve</span>
+    <span class="no-margin" ng-if="$ctrl.sceneStatusCounts.APPROVED">
+      <ng-pluralize count="$ctrl.sceneStatusCounts.APPROVED"
+                    when="{'one': '1 scene',
+                          'other': '{} scenes'
+                          }">
+      </ng-pluralize>
+      <strong class="color-primary">approved</strong>
+    </span>
+    <span class="no-margin" ng-if="$ctrl.sceneStatusCounts.APPROVED && $ctrl.sceneStatusCounts.REJECTED">&nbsp;and&nbsp;</span>
+    <span class="no-margin" ng-if="$ctrl.sceneStatusCounts.REJECTED">
+      <ng-pluralize count="$ctrl.sceneStatusCounts.REJECTED"
+                    when="{'one': '1 scene',
+                          'other': '{} scenes'
+                          }">
+      </ng-pluralize>
+      <strong class="color-danger">rejected</strong>
+    </span>
+  </div>
+</div>
+<div class="sidebar-scrollable list-group">
+  <rf-scene-item
+      ng-click="$ctrl.openSceneDetailModal(scene)"
+      scene="scene"
+      is-disabled="$ctrl.isInProject(scene)"
+      on-select="$ctrl.setSelected(scene, selected)"
+      class="selectable"
+      ng-mouseenter="$ctrl.setHoveredScene(scene)"
```

Comment:
It looks like this isn't implemented in the controller (I noticed it wasn't working when testing), but it would be nice to have; it will be more difficult for users to approve/reject scenes without being able to see thumbnails on the map.